Add Knighty-knight

AliNisarAhmed opened the pull request.

dwayne commented:
@AliNisarAhmed Thanks for your contribution. A few things:

1. Please remove the Elm debugger tool from the screenshot.
2. Only add your data to the `projects.json` file and don't change anything else in the file. However, you can submit a separate PR for the other changes if you wish.

AliNisarAhmed committed:
Add Knighty-knight

AliNisarAhmed commented:
@dwayne I have updated the screenshot, and although the other changes were minor formatting fixes (which seemed like the correct thing to do), I have removed those changes.

dwayne commented:
@AliNisarAhmed This is great! Thanks for taking the time to make the changes to your PR.

You are free to submit another PR with the minor formatting fixes. If not, I will make them myself before I deploy the changes on Sunday. Thanks for pointing them out.
